### DISCUSSION TITLE
Differences between Flexify and Massive, and how to import custom plans/sets?

### DISCUSSION THREAD

**Discussion by coreystone** [brandonp2412/Flexify#126] [Q&A] [answered]
Hi, I've been using Flexify but recently discovered [Massive](https://github.com/brandonp2412/Massive) and I couldn't find any announcement or discussion about the differences. I can see that Massive is an earlier version that seems to no longer be maintained. Is there any reason to use it over Flexify?

Also, I am trying to import custom sets into Flexify by editing graphs.csv but haven't had any success. I'm trying to manually insert the sets into the CSV which were recorded in a notebook, not Massive. I am using the proper format for date `created `and `hidden` is set to false. When I import the graphs.csv file into Flexify, all my graphs disappear. Do I need to create the exercises in a plan first before importing sets for them? I've also had trouble importing plans.csv in the same way.

I wonder if this has something to do with the `id` value I am setting, which I tried to just set in descending order for all sets. Do `id` and date `created` need to be in a particular order? That is, should the earliest date have the smallest id? Thanks for the great app, excited to see what's in store in the future. 💯

**Comment by brandonp2412** [comment, marked as answer]
Massive is still maintained I'm only going to fix bugs on it occasionally. Main difference between the two is drawer layout vs tabs.

Any CSV exporting errors should be logged with relevant [adb logs](https://developer.android.com/tools/logcat)
